.vscode/settings.json

```diff
@@ -13,5 +13,6 @@
 
     "flow.enabled": true,
     "flow.useNPMPackagedFlow": true,
-    "javascript.validate.enable": false
+    "javascript.validate.enable": false,
+    "jest.pathToConfig": "configs/jest.config.json"
```

Comment:
This option is used for https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest ?

Reply:
🤖 Affirmative
It's quite useful BTW